src/AppKit/Enums.cs

```diff
@@ -2154,10 +2154,191 @@ public enum NSPopUpArrowPosition : ulong {
 		Bottom
 	}
 
-	public static class NSFileTypeForHFSTypeCode {
-		public static readonly string ComputerIcon = "root";
-		public static readonly string DesktopIcon = "desk";
-		public static readonly string FinderIcon = "FNDR";
+	// FileType 4cc values to use with NSFileTypeForHFSTypeCode.
+	public enum HFSTypeCode: uint
+	{
+		/* Generic Finder icons */
+		ClipboardIcon                 = 0x434C4950,   //'CLIP'
+		ClippingUnknownTypeIcon       = 0x636C7075,   //'clpu'
+		ClippingPictureTypeIcon       = 0x636C7070,   //'clpp'
+		ClippingTextTypeIcon          = 0x636C7074,   //'clpt'
+		ClippingSoundTypeIcon         = 0x636C7073,   //'clps'
+		DesktopIcon                   = 0x6465736B,   //'desk'
+		FinderIcon                    = 0x464E4452,   //'FNDR'
+		ComputerIcon                  = 0x726F6F74,   //'root'
+		FontSuitcaseIcon              = 0x4646494C,   //'FFIL'
+		FullTrashIcon                 = 0x66747268,   //'ftrh'
+		GenericApplicationIcon        = 0x4150504C,   //'APPL'
+		GenericCDROMIcon              = 0x63646472,   //'cddr'
+		GenericControlPanelIcon       = 0x41505043,   //'APPC'
+		GenericControlStripModuleIcon = 0x73646576,   //'sdev'
+		GenericComponentIcon          = 0x74686E67,   //'thng'
+		GenericDeskAccessoryIcon      = 0x41505044,   //'APPD'
+		GenericDocumentIcon           = 0x646F6375,   //'docu'
+		GenericEditionFileIcon        = 0x65647466,   //'edtf'
+		GenericExtensionIcon          = 0x494E4954,   //'INIT'
+		GenericFileServerIcon         = 0x73727672,   //'srvr'
+		GenericFontIcon               = 0x6666696C,   //'ffil'
+		GenericFontScalerIcon         = 0x73636C72,   //'sclr'
+		GenericFloppyIcon             = 0x666C7079,   //'flpy'
+		GenericHardDiskIcon           = 0x6864736B,   //'hdsk'
+		GenericIDiskIcon              = 0x6964736B,   //'idsk'
+		GenericRemovableMediaIcon     = 0x726D6F76,   //'rmov'
+		GenericMoverObjectIcon        = 0x6D6F7672,   //'movr'
+		GenericPCCardIcon             = 0x70636D63,   //'pcmc'
+		GenericPreferencesIcon        = 0x70726566,   //'pref'
+		GenericQueryDocumentIcon      = 0x71657279,   //'qery'
+		GenericRAMDiskIcon            = 0x72616D64,   //'ramd'
+		GenericSharedLibaryIcon       = 0x73686C62,   //'shlb'
+		GenericStationeryIcon         = 0x73646F63,   //'sdoc'
+		GenericSuitcaseIcon           = 0x73756974,   //'suit'
+		GenericURLIcon                = 0x6775726C,   //'gurl'
```

Comment:
`Url`
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers